resources/assets/lib/quick-search/main.tsx

```diff
@@ -105,18 +121,21 @@ const otherModes: ResultMode[] = ['forum_post', 'wiki_page'];
 
     return (
       <div className='quick-search-items'>
-        {this.props.worker.searchResult.beatmapset.beatmapsets.map((beatmapset) => {
+        {this.props.worker.searchResult.beatmapset.beatmapsets.map((beatmapset, idx) => {
           return (
             <div key={beatmapset.id} className='quick-search-items__item'>
-              <Beatmapset beatmapset={beatmapset} />
+              <Beatmapset
+                beatmapset={beatmapset}
+                active={this.boxIsActive(Section.Beatmapset, idx)}
+              />
```

Comment:
pass style modifiers as `props` instead of an active flag if the component is only using the flag to set a style modifier (see `user-card.tsx`)